examples/bri-3/LogConfig.ts

```diff
@@ -0,0 +1,7 @@
+import { Logger } from 'tslog';
```

Comment:
this file is now obsolete and can be removed completely

Reply:
Updated